### PR TITLE
PowerTubeConfigure Feature implementented and tests modified/added

### PR DESCRIPTION
**Pull request by skragh**

### CODE REVIEW

**Comment by skragh** [comment]
Ready for review

File: Microwave.Classes/Boundary/PowerTube.cs [hunk 1]
```diff
@@ -9,24 +9,29 @@ public class PowerTube : IPowerTube
 
         private bool IsOn = false;
 
-        public PowerTube(IOutput output)
+        private int MaxPower;
+        private int CurrentPower = 0;
+
+        public PowerTube(IOutput output, int maxPower)
```
Comment (Omarley7) [inline Microwave.Classes/Boundary/PowerTube.cs:15]: Nice touch, maybe leave a default version for support of legacy applications?

File: Microwave.Classes/Boundary/PowerTube.cs [hunk 2]
```diff
@@ -9,24 +9,29 @@ public class PowerTube : IPowerTube
...
+                throw new ArgumentOutOfRangeException("power", power, $"Must be between 1% and 100% (incl.)");
             }
 
             if (IsOn)
             {
                 throw new ApplicationException("PowerTube.TurnOn: is already on");
             }
-
-            myOutput.OutputLine($"PowerTube works with {power}");
+            //calculate power to use
```
Comment (Omarley7) [inline Microwave.Classes/Boundary/PowerTube.cs:32]: Good thing with comments, nice work

File: Microwave.Classes/Controllers/UserInterface.cs [hunk 3]
```diff
@@ -56,7 +58,7 @@ public void OnPowerPressed(object sender, EventArgs e)
                     myState = States.SETPOWER;
                     break;
                 case States.SETPOWER:
-                    powerLevel = (powerLevel >= 700 ? 50 : powerLevel+50);
+                    powerLevel = (powerLevel >= maxPower ? 10 : powerLevel+10);  //power increases by 10% at a time, ranges 10,20,30% etc of max power.
```
Comment (Omarley7) [inline Microwave.Classes/Controllers/UserInterface.cs:61]: I love inline if statements keep it up

File: MicrowaveOven.sln [hunk 4]
```diff
@@ -9,8 +9,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microwave.Test.Unit", "Micr
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microwave.App", "Microwave.App\Microwave.App.csproj", "{DEA97537-CDF7-4246-AB7A-B6C08D125768}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microwave.Test.Integration", "Microwave.Test.Integration\Microwave.Test.Integration.csproj", "{991A3729-88A6-44A4-AB3A-5216B944575C}"
```
Comment (Omarley7) [inline MicrowaveOven.sln:12]: Fint jeg gir mig. Lad os slette det.

File: Microwave.Test.Unit/UserInterfaceTest.cs [hunk 5]
```diff
@@ -61,45 +61,45 @@ public void DoorOpen_DoorClose_LightOff()
...
-            display.Received(1).ShowPower(Arg.Is<int>(700));
+            display.Received(1).ShowPower(Arg.Is<int>(100));
         }
 
         [Test]
-        public void Ready_15PowerButton_PowerIs50Again()
+        public void Ready_11PowerButton_PowerIs10Again()
         {
-            for (int i = 1; i <= 15; i++)
+            for (int i = 1; i <= 11; i++)
```
Comment (Omarley7) [inline Microwave.Test.Unit/UserInterfaceTest.cs:96]: Noice

**Review by Omarley7 — APPROVED** [review]
Very good sir. I approve this messages.

**Pull request merged**